config.json

```
@@ -1879,6 +1879,23 @@
           "conditionals-if"
         ],
         "difficulty": 8
+      },
+      {
+        "slug": "sgf-parsing",
+        "name": "SGF Parsing",
+        "uuid": "0d6325d1-c0a3-456e-9a92-cea0559e82ed",
+        "practices": ["refactoring", "trees", "switch-case", "conditionals-if", "strings"],
```

```suggestion
        "practices": [],
```

This array should contain concepts that exist in the track, e.g `refactoring` doesn't exist as a concept in the track.

This array makes the exercise appear next to the concept exercises that teach the concepts listed here. However, since this is a hard exercise, let's just leave this empty for now.

---

Thanks for clarifying this. I was a bit confused about those two fields.